### ISSUE TITLE
Hochschild, Quillen or Triple (co)homology 

### ISSUE THREAD

**Issue by nasief** [homalg-project/homalg_project#548]
Hi guys

I am a newbi to the whole GAP thing, I am wondering if there is a way to calculate the homologies of Hochschild, Quillen or Triple?

**Comment by mohamed-barakat** [comment]
For the Hochschild cohomology, one needs the so-called envelope algebra $A^e$ to be a computable ring in the sense defined in this [paper](https://arxiv.org/abs/1003.1943). For all algebras $A$ for which $A^e$ is computable one can compute the Hochschild cohomology, yes.

**Comment by mohamed-barakat** [comment]
Which algebra $A$ are you interested in?

**Comment by nasief** [comment]
I am interested in polynomial algebras over prime fields FF_p[[x]] modulo some dividing power x^n

**Comment by mohamed-barakat** [comment]
Yes, computing the Hochschild cohomology for these algebras is very simple. Do you have specific examples you want to compute?

**Comment by nasief** [comment]
What I am looking for is I want to calculate this kind of homology but there are some symmetry conditions imposed on the chains , they are symmetric in some sense

Here is the link to that article

https://www.ams.org/journals/tran/1962-104-02/S0002-9947-1962-0142607-6/S0002-9947-1962-0142607-6.pdf

I want to consider say polynomial algebra over F_3 modulo  x^n with coefficients in F_3 subject to those symmetry conditions which are described for dimension 1,2 and 3

**Comment by mohamed-barakat** [comment]
I didn't have time to look at the new commutative algebra cohomology described in the paper. Hochschild cohomology for the cases you describe is easy if this is would be a good starting point.

**Comment by nasief** [comment]
I have a good sense of Hochschild. But what I am looking for are those other notions of cohomology.
